pkg/api/cache.go

```diff
@@ -19,19 +19,27 @@ import (
 	"github.com/renstrom/fuzzysearch/fuzzy"
 )
 
+const ( // cache fields
+	REGION   = iota
```

Comment:
exported const REGION should have comment (or a comment on this block) or be unexported